ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -550,6 +550,23 @@ interface {{ ethernet_interface.name }}
 {%     if ethernet_interface.poe.legacy_detect is arista.avd.defined(true) %}
    poe legacy detect
 {%     endif %}
+{%     if ethernet_interface.switchport is arista.avd.defined %}
+{%         if ethernet_interface.switchport.port_security.enable is arista.avd.defined(true) %}
```

Comment:
Are all the other commands visible on EOS even if the main command is not configured? Depending on this we should nest the remaining commands under the enabled here or not.

Reply:
yes, I can see `switchport port-security mac-address maximum 4` and others, without configuring `switchport port-security`